identity-iota/src/client/client_builder.rs

```diff
@@ -41,9 +43,14 @@ impl ClientBuilder {
     self
   }
 
+  pub fn node_sync_disabled(mut self) -> Self {
```

Comment:
`pub fn node_sync_enabled(mut self, value: bool) -> Self` leaves much more flexibility for the caller